lib/buildenvsetup/ceconan.js

```diff
@@ -91,8 +91,9 @@ export class BuildEnvSetupCeConanDirect extends BuildEnvSetupBase {
         });
     }
 
-    async downloadAndExtractPackage(downloadPath, packageUrl) {
+    async downloadAndExtractPackage(libId, version, downloadPath, packageUrl) {
```

Comment:
do we use the libId and version? Or is that coming soon? (I just noticed this is WIP! sorry)